drivers/ieee802154/ieee802154_b9x_frame.c

```diff
@@ -603,7 +603,15 @@ b9x_ieee802154_get_data(const uint8_t *payload,
  * Cryptography functionality
  */
 
+#ifdef CONFIG_IEEE802154_2015
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <aes.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <ske.h>
+#include <ske_portable.h>
+#else
```

Comment:
we haven't other SoCs in B9X family, why we need here #else  ?
looks like can be removed

Reply:
OK, i will check with zhenghuan , seems can be delete.

Reply:
already update code